dm/dm.go

```diff
@@ -204,28 +202,42 @@ func describeType(args []string) {
 		usage()
 	}
 
+	tUrl := getTypeUrl(args[1])
+	schemaUrl := tUrl + ".schema"
+	fmt.Println(callHttp(schemaUrl, "GET", "get schema for type ("+tUrl+")", nil))
+}
+
+func getTypeUrl(tName string) string {
 	var tUrl string
 
-	if strings.HasPrefix(args[1], "http://") || strings.HasPrefix(args[1], "https://") {
+	if isHttp(tName) {
 		// User can pass raw URL to template.
-		tUrl = args[1]
+		tUrl = tName
```

Comment:
why not get rid of tUrl variable and here just return tName and on L223,
return getDownLoadUrl(*t)